Merge my Blog and the picture

ShahaamMarzook opened the pull request:
Please read and understand everything below
**Do not delete any text other than where you are instructed.**

**Students: If one of them is applicable to you. Please check it.**

Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [ ] Read and understood (see CONTRIBUTING.md)
- [ ] Included a Preview link and screenshot showning after and before the changes.
- [ ] Images are `240 x 240` [w x h].
- [ ] Included a description of change below.
- [ ] Squashed the commits.

# Changes done in this Pull Request

- If your change will be reflected on the website, please provide a **Test-Link** (**Hint : `gh-pages`**)
- Fixes`#<Issue no.>`

ShahaamMarzook committed:
Uploaded my blog

ShahaamMarzook committed:
Blog pic uploaded

nikit19 reviewed (changes requested):
Looks good! Just resolve the conflict :+1:

isuruAb reviewed (commented):
please resolve conflicts.

ShahaamMarzook reviewed (commented):
Accpt this So,that my task would be apporved

ShahaamMarzook commented:
Accept my pr please

ShahaamMarzook committed:
Merge branch 'gh-pages' into gh-pages

ShahaamMarzook commented:
please Merge my pull request

ShahaamMarzook commented:
@nikit19 please approve this now ASAP
I'm tring to fiish this take for the past two days, willing to do the other tasks.Please approve this as well as my task  https://codein.withgoogle.com/dashboard/task-instances/5733699029041152/

ShahaamMarzook committed:
update blog.yml

updated My blog
img: Blog pic.jpeg updated to img:Shahaam.jpeg

ShahaamMarzook committed:
Shahaam.jpeg uploded for Shahaam Marzook blog

ShahaamMarzook committed:
Add files via upload

added a pic

ShahaamMarzook committed:
Merge pull request #1 from ShahaamMarzook/ShahaamMarzook-patch-1

Add files via upload

ShahaamMarzook committed:
Delete shahaam.jpeg

Sorry uploaded the pic in the repository

ShahaamMarzook committed:
Added Shahaam.jpeg

removed Shahaam.jpeg from repository and deleted it and added it again in this where this pic is supposed to be uploaded.
Please merge this.And help me in passing this blogging task in GCI17.
With Respect,
Shahaam Marzook

ShahaamMarzook commented:
Mentors,
   Please approve this and merge this , because I'm trying to pass this for the past 02 days.And help me in completing the task ;
 https://codein.withgoogle.com/dashboard/task-instances/5733699029041152/

ShahaamMarzook committed:
Delete Blog pic.jpeg

Added a new pic Shahaam.jpeg

ShahaamMarzook committed:
dlete Shahaam.jpeg

not in 240x240 pic
updated a 240x240 pic

ShahaamMarzook committed:
Uploaded a Shahaam.jpg

updated Shahaam.jpg

ShahaamMarzook commented:
Mentors,
Please approve this and merge this , because I'm trying to pass this for the past 02 days.And help me in completing the task ;  😄 
https://codein.withgoogle.com/dashboard/task-instances/5733699029041152/?sp-page=2

ShahaamMarzook commented:
@nikit19 So,have I merged my blog into the main repository.
😮

nikit19 commented on _data/blogs.yml:
```diff
@@ -273,6 +273,11 @@
 
+- url: https://workingwithfossasiawordpress.wordpress.com/2018/01/06/participation-in-open-source-community/
+  title: Participation in Open source community
+  img: Shahaam.jpeg
```
The image that you uploaded and wrote here don't have the same name.
One ends with jpg and the other with jpeg.
Please make it same

nikit19 reviewed (changes requested):
Please make this change

realslimshanky reviewed (changes requested):
Please squash your commits.

ShahaamMarzook commented:
I have uploaded the picture in the img/blog As Shahaam.jpg 
Please review my task and accept the pull request as well.   😄 
https://codein.withgoogle.com/dashboard/task-instances/5733699029041152/

With respect,
Shahaam Marzook

nikit19 commented:
But you wrote 'Shahaam.jpeg' in blogs.yml
The name is different,please make it same both the places

ShahaamMarzook committed:
Update blog.yml

harshit98 reviewed (approved):
Good work!
